gridmenuview/src/main/java/com/bcc/gridmenuview/GridMenu.java

```diff
@@ -104,6 +119,16 @@ public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull R
                 int emptySpacePx = parent.getMeasuredWidth() - spanCount * viewWidth;
                 outRect.left = outRect.right = emptySpacePx / (2 * spanCount);
             }
+
+            ImageView menuItemView = view.findViewById(R.id.iv_image);
+            ImageView menuItemOverlay = view.findViewById(R.id.iv_image_overlay);
+            if (shapeItem == ShapeElement.SQUARE.label) {
```

Comment:
better if you use switch-case rather than if-else

Reply:
and also you can separate this checking shape functionality to other methods. I suggest you to read regarding DRY Principle